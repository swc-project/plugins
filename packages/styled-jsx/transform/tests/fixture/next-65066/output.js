import _JSXStyle from "styled-jsx/style";
import React from "react";
// This should error with https://nextjs.org/docs/messages/nested-styled-jsx-tags
// but instead it crashes SWC.
export default function Home() {
    return <div className={"jsx-915beb22be3340e9"}>
            <div className={"jsx-915beb22be3340e9"}>
                <div className={"jsx-915beb22be3340e9" + " " + "text animated"}>Text</div>

                <_JSXStyle id={"915beb22be3340e9"}>{".text.jsx-915beb22be3340e9:hover{color:#ff0!important}"}</_JSXStyle>
            </div>

            <style jsx>{`
        .text:hover {
          color: yellow !important;
        }
      `}</style>
        </div>;
}
