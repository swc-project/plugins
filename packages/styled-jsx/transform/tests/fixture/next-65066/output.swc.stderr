  x Detected nested styled-jsx tag.
  | Read more: https://nextjs.org/docs/messages/nested-styled-jsx-tags
    ,-[input.js:22:1]
 21 |     
 22 | ,->             <style jsx>{`
 23 | |           .text:hover {
 24 | |             color: yellow !important;
 25 | |           }
 26 | `->       `}</style>
 27 |             </div>
    `----
